.github/PULL_REQUEST_TEMPLATE/bug.md

```diff
@@ -0,0 +1,19 @@
+**Related Issue:** GH-
```

Comment:
It would be helpful to state here as well that [//]-blocks do not have to be removed e.g. [//]: # (Lines starting with [//]: are considered to be comments. You do not have to delete those lines, they are not rendered upon release.)